src/Symfony/Bridge/ProxyManager/Legacy/ProxiedMethodReturnExpression.php

```diff
@@ -51,9 +51,6 @@ public static function generate(string $returnedValueExpression, ?\ReflectionMet
         if ($functionLoader instanceof ClassLoader) {
             return $functionLoader;
         }
-        if ($functionLoader instanceof \Symfony\Component\Debug\DebugClassLoader) {
-            return $getComposerClassLoader($functionLoader->getClassLoader());
-        }
         if ($functionLoader instanceof \Symfony\Component\ErrorHandler\DebugClassLoader) {
```

Comment:
we can use a `use` statement in master now

Reply:
done